src/internal/_equals.js

```diff
@@ -1,114 +1,161 @@
+var _containsWith = require('./_containsWith');
```

Comment:
Can you move this below the next line? The requires need to be in alphabetical order..